google-cloud-contrib/google-cloud-logging-adapters/cloud-logging-over-slf4j/README.md

```diff
@@ -0,0 +1,68 @@
+## Slf4j Adapter for Google Cloud Logging
+
+Built with [Slf4j](https://www.slf4j.org/) v1.7.21 and 
+[Google cloud logging library](https://github.com/GoogleCloudPlatform/google-cloud-java/tree/v0.9.2/google-cloud-logging)
+v0.9.2
+
+Adapter library to enable Slf4j framework to log to [Google Cloud logging](https://cloud.google.com/logging/docs/view/logs_viewer_v2).
+
+
+## Configuration and setup
+Install [Apache Maven](https://maven.apache.org/) <p>
+`mvn clean install`
+
+If you are using Maven, add this to your pom.xml file
+```xml
+<dependency>
+  <groupId>com.example</groupId>
+  <artifactId>cloud-logging-over-slf4j</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+    
+
+### Configuration file
+Set system property or environment variable `CLOUD_LOGGING_CONFIG` to absolute path of .yaml configuration file. <p>
+Example config file : [google-cloud-logging.yaml](src/test/resources/google-cloud-logging.yaml)<p>
+#### Top level fields
+- `destination` (optional) : defaults to direct writes to cloud via API. <p>
+Allowed values : <p>
+ -`"stdout"` : Prints logs as json to stdout
+ -`"fluentd"` logs to fluentd forwarding port (`localhost:24224`)
+ 
+- `loggers` (list) : (optional) (uses default configuration if none specified)
+  ##### Required fields for custom loggers : 
+  - `name` ["default" or package/class name ]
+  ##### Optional fields : 
```

Comment:
It doesn't look like this line is being formatted correctly in the rendered file.